website/docs/r/kinesis_metrics_source.html.markdown

```diff
@@ -128,4 +128,282 @@ HTTP sources can be imported using the collector name and source name (`collecto
 terraform import sumologic_kinesis_metrics_source.test my-test-collector/my-test-source
 ```
 
+## Full Example (Including terraform for AWS asset creation)
+```hcl
+terraform {
+  required_providers {
+    sumologic = {
+      source = "sumologic/sumologic"
+    }
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+}
+
+provider "sumologic" {}
+provider "aws" {}
+
+locals {
+  account_id = ""
+  aws_access_key = ""
+  aws_secret_key = ""
+
+  description = "update your terraform description here"
+  identifier = "SumologicMetricStream"
+
+  region = "us-west-2"
+
+  tagfilters = [
+    { type = "TagFilters", namespace = "AWS/ApplicationELB", tags = ["Deployment=prod"] },
+  ]
+
+}
+
+resource "sumologic_collector" "collector_for_kinesis_metrics" {
+  name = "AWS Metrics via Kinesis"
+}
+
+resource "sumologic_kinesis_metrics_source" "kinesis_source" {
+  name          = "CloudWatch Metrics via Kinesis"
+  description   = "Description for Sumologic source"
+  category      = "aws/cloudwatch"
+  content_type  = "KinesisMetric"
+  collector_id  = sumologic_collector.collector_for_kinesis_metrics.id
+
+  authentication {
+    type = "S3BucketAuthentication"
+    access_key = local.aws_access_key
+    secret_key = local.aws_secret_key
+  }
+
+  path {
+    type = "KinesisMetricPath"
+
+    dynamic "tag_filters" {
+      for_each = local.tagfilters
+      content {
+        type      = tag_filters.value.type
+        namespace = tag_filters.value.namespace
+        tags      = tag_filters.value.tags
+      }
+    }
+  }
+}
+
+// ------------------------------------ AWS Kinesis part
+
+resource "aws_cloudwatch_metric_stream" "main" {
+  name          = local.identifier
+  role_arn      = aws_iam_role.metric_stream_to_firehose.arn
+  firehose_arn  = aws_kinesis_firehose_delivery_stream.kinesis_stream.arn
+  output_format = "opentelemetry0.7"
+
+//  include_filter {
+//    namespace = "AWS/ApplicationELB"
+//  }
+//  include_filter {
+//    namespace = "AWS/DynamoDB"
+//  }
```

Comment:
does this need to be uncommented?

Reply:
I added comment to explain the lines, i.e. I think adding commented example is good in this case